website/docusaurus.config.js

```diff
@@ -117,6 +118,10 @@ module.exports = {
               label: 'Blog',
               href: 'https://configcat.com/blog/',
             },
+            {
+              label: 'Limitations',
```

Comment:
I think the Limitations is a bit to broad. Maybe Subscription limitations or Plan limitations would be better.